README.md

```diff
@@ -1,15 +1,21 @@
-# Idris Vimscript
+# Idris in Vimscript!
```

Comment:
I'd rather keep the (somewhat unclear) bidirectional heading for now. It seems it could just as well have been "Vimscript in Idris". Hope that makes sense? 😄

Reply:
Sure, hadn't considered that.